openpype/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -189,6 +189,12 @@ def get_openpype_executable():
     exe_list = config.GetConfigEntryWithDefault("OpenPypeExecutable", "")
     dir_list = config.GetConfigEntryWithDefault(
         "OpenPypeInstallationDirs", "")
+
+    # clean '\ ' for MacOS pasting
+    if exe_list:
+        exe_list = exe_list.replace("\\ ", " ")
+    if dir_list:
+        dir_list = dir_list.replace("\\ ", " ")
```

Comment:
If it's Mac specific it should happen on Mac. And I think the check of value is not needed as it should return always string.
```suggestion
    # clean '\ ' for MacOS pasting
    if platform.system().lower() == "darwin":
        exe_list = exe_list.replace("\\ ", " ")
        dir_list = dir_list.replace("\\ ", " ")
```